frontend/app/views/spree/users/show.html.erb

```diff
@@ -13,7 +13,7 @@
 
       <div class="d-lg-inline-block account-page-user-info-item-store-credits">
         <p class="my-0 text-uppercase text-center">
-          <%= Spree.t(:store_credit_name) %>: <b class="account-page-user-info-item-store-credits-amount"><%= Spree::Money.new(@user.total_available_store_credit(current_currency), currency: current_currency).to_s.gsub('.00', '') %></b>
+          <%= Spree.t(:store_credit_name) %>: <b class="account-page-user-info-item-store-credits-amount"><%= Spree::Money.new(@user.total_available_store_credit(current_currency, current_store), currency: current_currency).to_s.gsub('.00', '') %></b>
```

Comment:
It's ok, although later it could not call Spree::Money in here and use this
https://github.com/spree/spree/pull/11142/files#diff-cbe892c2336366a40b1f8a6f9bd28ebf8e7d1d1eb5f4a3223e475f69b841f56fR33
We need just to merge the mentioned PR.

Reply:
@tomurb Yes, I don't wanted to duplicate you work in this pr.